cli/tests/program.rs

```diff
@@ -1347,8 +1369,9 @@ fn test_cli_program_close_program() {
     assert_eq!(programdata_lamports, recipient_account.lamports);
 }
 
-#[test]
-fn test_cli_program_extend_program() {
+#[test_case(true; "Skip Preflight")]
+#[test_case(false; "Dont skip Preflight")]
+fn test_cli_program_extend_program_skip_preflight(skip_preflight: bool) {
```

Comment:
For the sake of faster tests, we can probably remove the test-case here, since the interesting behavior is for the program deployment, and it's covered elsewhere